driver-mct/main/cime_comp_mod.F90

```diff
@@ -4831,9 +4831,6 @@ subroutine cime_run_calc_budgets2(in_cplrun)
        if (ice_present) then
           call seq_diag_ice_mct(ice(ens1), fractions_ix(ens1), infodata, do_i2x=.true.)
        endif
-       if (glc_present) then
-          call seq_diag_glc_mct(glc(ens1), fractions_gx(ens1), infodata, do_g2x=.true.)
```

Comment:
@jonbob , do you have any idea why g2x budgets were previously done after components are run?

Reply:
I can also run some tests to see if / how the budgets change if we move the location where glc budgets are calculated form budgets 1 to budgets 2.

Reply:
I think usually we want the updated values for g2x after glc has run -- but we can test that